### PR TITLE
Add missing styles for PLP and PDP

### PR DESCRIPTION
**Pull request by kschmidtdev**
Our Home template looks great, but the PLP and PDP have giant icons and the nav bar doesn't look right. This PR fixes that.

~~**JIRA**: (link to JIRA ticket)~~

## Changes
- Add missing style includes to the stylesheets of the PLP and PDP

## How to test-drive this PR
- Run `npm test`
- Run `npm run dev`
- Check http://localhost:3000, http://localhost:3000/potions.html, http://localhost:3000/eye-of-newt.html

### CODE REVIEW

File: amp/app/containers/product-details/container.scss [hunk 1]
```diff
@@ -17,6 +25,8 @@
 
 @import '../../components/amp-lightbox/base';
 @import '../../components/amp-lightbox/theme';
+@import '../../components/sheet/base';
```
Comment (yourpalsonja) [inline amp/app/containers/product-details/container.scss]: Do we maybe want to add `Sheet` to `/app/_base.scss`, since it is getting used on every page?

Comment (kschmidtdev) [inline amp/app/containers/product-details/container.scss]: Sounds like a good idea, your thoughts on this @cole-sanderson ?

Comment (cole-sanderson) [inline amp/app/containers/product-details/container.scss]: Yeah, that will be good idea 👍

**Comment by cole-sanderson** [comment]
Looks good 👍

**Comment by kschmidtdev** [comment]
@cole-sanderson can you give this one more look? Thanks!

**Comment by cole-sanderson** [comment]
Looks good 👍

**Pull request merged**